public/app/plugins/datasource/prometheus/querybuilder/components/wizarDS/WizarDS.tsx

```diff
@@ -75,31 +53,24 @@ export const WizarDS = (props: WizarDSProps) => {
       {/* WizarDS */}
       {/* header */}
       <div className={styles.header}>
-        <h3>WizarDS, aka the Wizard for Data Sources</h3>
+        <h3>Query wizard</h3>
         <Button icon="times" fill="text" variant="secondary" onClick={closeDrawer} />
       </div>
       {/* Starting message */}
       <div>
         <div className={styles.iconSection}>
-          <img src={AI_Logo_color} alt="AI logo color" /> WizarDS
+          <img src={`public/img/ai-icons/AI_Logo_color.svg`} alt="AI logo color" /> Query wizard
         </div>
         {state.showStartingMessage ? (
           <>
             <div className={styles.textPadding}>
-              The WizarDS can take you on a journey through the Prometheus UI using AI
+              The Query wizard can take you on a journey through the Prometheus UI using AI
             </div>
-            <div className={styles.textPadding}>The WizarDS will connect to OpenAI using your API key.</div>
-            {/* <div className={styles.dataList}>
-              <ul>
-                <li>Metrics</li>
-                <li>Labels</li>
-                <li>Metrics metadata</li>
-              </ul>
-            </div> */}
+            <div className={styles.textPadding}>The Query wizard will connect to OpenAI using your API key.</div>
             <div className={styles.textPadding}>Check with OpenAI to understand how your data is being used.</div>
             <div>
-              The WizarDS information comes from Grafana docs and is interpreted by ChatGPT when you enter a prompt.
-              Please be aware that ChatGPT can hallucinate.
+              The Query wizard information comes from Grafana docs and is interpreted by ChatGPT when you enter a
```

Comment:
```suggestion
              The Query wizard information comes from the Grafana documentation and is interpreted by ChatGPT when you enter a
```

Reply:
Sorry I just noticed this! I will add this to my current branch. Language is good!